addons/osio-addon/src/main/java/io/fabric8/launcher/osio/providers/DependencyParamConverterProvider.java

```diff
@@ -0,0 +1,28 @@
+package io.fabric8.launcher.osio.providers;
+
+import org.apache.maven.model.Dependency;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.inject.Instance;
+import javax.inject.Inject;
+import javax.ws.rs.ext.ParamConverter;
+import javax.ws.rs.ext.ParamConverterProvider;
+import javax.ws.rs.ext.Provider;
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Type;
+
+@Provider
+@ApplicationScoped
+public class DependencyParamConverterProvider implements ParamConverterProvider {
+    @Inject
+    private Instance<DependencyParamConverter> dependencyParamConverter;
```

Comment:
You can @Inject DependencyParamConverter directly here, since they share the same scope